Detectors/HMPID/workflow/README.md

```diff
@@ -56,7 +56,8 @@ Data processor options: HMP-DataDecoder:
   --root-file arg (=/tmp/hmpRawDecodeResults)
                   Name of the Root file with the decoding
                   results.
-  --fast-decode   Use the fast algorithm. (error 0.8%
+  --fast-decode   Use the fast algorithm. (error 0.8%)
+  --get-results   Switch on the output of statistic result files
```

Comment:
Thx for the PR, wouldn't --get-results-statistics be a better name for the option?`Just --get-results seems a bit misleading to me.

Reply:
done.
Now I need to redo the PR or not ?

Reply:
What do you mean be redo? You have pushed to the branch of the PR, so the PR was automatically updated with your commit.